### PR TITLE
KULRICE-14145 : Maven Upgrade in support of Java 8 Part 2

### PR DESCRIPTION
**Pull request by cwade179**
Maven libraries Upgrade without the groovy upgrade

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -125,7 +129,7 @@
     <displaytag.version>1.2</displaytag.version>
     <dom4j.version>1.6.1</dom4j.version>
     <dwr.version>3.0.RC2</dwr.version>
-    <eclipselink.version>2.5.2</eclipselink.version>
+    <eclipselink.version>2.6.0-M3</eclipselink.version>
```
Comment (cniesen) [inline pom.xml]: As discussed during a standup, it's OK to upgrade eclipselink to a milestone version. Please add a jira to upgrade it to the release version before we release 2.6.0.

**Comment by cniesen** [comment]
This pull request replaces pull request https://github.com/kuali/rice/pull/18, right?  if so can you close the pr #18.

**Comment by cniesen** [comment]
looks good

**Comment by sonamuthu** [comment]
looks like the changes are causing the unit tests to break on ci

**Pull request merged**